site/src/pages/docs/server-multipart.mdx

```diff
@@ -0,0 +1,145 @@
+# Handling a multipart request
+
+Armeria provides <type://Multipart> encoder and decoder built on top of 
+[Reactive Streams](https://www.reactive-streams.org/).
+
+## Building a multipart request
+
+A <type://Multipart> consists of multiple <typeplural://BodyPart>.
+Each <type://BodyPart> is divided into a headers and a body. A <type://BodyPart> headers can be created simply 
```

Comment:
since headers is plural

```suggestion
Each <type://BodyPart> is divided into headers and a body. A <type://BodyPart> headers can be created simply 
```